packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/FishjamClientInternal.kt

```diff
@@ -175,12 +198,21 @@ internal class FishjamClientInternal(
     }
   }
 
-  fun join(peerMetadata: Metadata = emptyMap()) {
+  private suspend fun onDisconnected() {
```

Comment:
Because it would collide with cleaning up in the `fun leave()` when we leave and clean up everything completely. Here we don't want to clean up everything, just "prepare" for reconnection

Reply:
this function name does not suggest that its preparing for reconnections. Now we have in several files and clasess functions like on disconeccted, disconnect, leave etc. Almost everyone of those functions does somthing different and it would be better to somewhere specify use of this function. It does not have to be another function name but at least comment with description.